src/Splitio/Services/Common/EventsManager.cs

```diff
@@ -0,0 +1,245 @@
+﻿using Splitio.Domain;
+using Splitio.Services.Logger;
+using Splitio.Services.Shared.Classes;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Splitio.Services.Common
+{
+    public class EventsManager : IEventsManager
+    {
+        private readonly ConcurrentDictionary<SdkEvent, Dictionary<string, object>> _activeEvents;
```

Comment:
These are the "subscriptions", right? does it make sense to rename to `_activeSubscriptions`?

Reply:
will do